js/private/node-patches/src/fs.ts

```diff
@@ -628,38 +674,11 @@ export const patcher = (fs: any = _fs, roots: string[]) => {
                 dirname == '/'
             ) {
                 // not a link
-                return false
+                return escapedHop
```

Comment:
this line of code stands up? Was returning `false` wrong here?

Reply:
Sorry the diff got a bit ugly there... I inlined the `_tryParent` method now that it is only called once. Previously it was:
```
if (!_tryParent()) {
    return escapedHop
}
```
That false case is now this line...

Reply:
Oh I see. 👍